app/test_main.py

```diff
@@ -1 +1,46 @@
-# write your code here
+import pytest
+from unittest import mock
+from app.main import can_access_google_page
```

Comment:
fix imports ordering according to PEP8 - https://peps.python.org/pep-0008/#imports